### PR TITLE
feat: add dotnet-sdk8-0-200

### PR DESCRIPTION
**Pull request by benjiro**
This PR adds support for .NET SDK 8.0.2 (8.0.200) https://dotnet.microsoft.com/en-us/download/dotnet/8.0

### CODE REVIEW

File: Casks/dotnet-sdk8-0-200.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+cask "dotnet-sdk8-0-100" do
```
Comment (isen-ng) [inline Casks/dotnet-sdk8-0-200.rb:1]: Cask name mismatches the filename and version name

**Comment by isen-ng** [comment]
Thank you for the PR, but I believe the cask name mismatch will cause CI to fail

File: Casks/dotnet-sdk8-0-200.rb (hunk 1 above)
Comment (benjiro) [reply Casks/dotnet-sdk8-0-200.rb]: oops my bad, copy and past mistake 😓

**Comment by isen-ng** [comment]
@benjiro Thank you very much for your contribution

**Pull request merged**